docs/docusaurus/docs/core/define_expectations/create_an_expectation.md

```diff
@@ -0,0 +1,178 @@
+---
+title: Create an Expectation
+description: Create and modify an Expectation in Python.
+---
+import PrereqPythonInstalled from '../_core_components/prerequisites/_python_installation.md';
+import PrereqGxInstalled from '../_core_components/prerequisites/_gx_installation.md';
+
+import StandardArgumentsTable from './_expectations/_standard_arguments_table.md';
+
+import TabItem from '@theme/TabItem';
+import Tabs from '@theme/Tabs';
+
+An Expectation is a verifiable assertion about your data. Expectations make implicit assumptions about your data explicit, and they provide a flexible, declarative language for describing expected behavior. They can help you better understand your data and help you improve data quality. 
+
+<h2>Prerequisites</h2>
+
+- <PrereqPythonInstalled/>.
+- <PrereqGxInstalled/>.
+
+<Tabs>
+
+<TabItem value="procedure" label="Procedure">
+
+1. Choose an Expectation to create.
+
+   GX comes with many built in Expectations to cover your data quality needs.  You can find a catalog of these Expectations in the [Expectation Gallery](https://greatexpectations.io/expectations/).  When browsing the Expectation Gallery you can filter the available Expectations by the data quality issue they address and by the Data Sources they support.  There is also a search bar that will let you filter Expectations by matching text in their name or docstring.
+
+   In your code, you will find the classes for Expectations in the `expectations` module:
+
+   ```python title="Python"
+   from great_expectations import expectations as gxe
+   ```
+
+2. Determine the Expectation's evaluation parameters
```

Comment:
"Evaluation parameters" is AFAIK not the right term here. That has basically been renamed "expectation parameters". I would just call these arguments/parameters, or something along those lines.